include/aspect/utilities.h

```diff
@@ -32,7 +32,6 @@
 #include <deal.II/dofs/dof_handler.h>
 #include <deal.II/fe/component_mask.h>
 
-#include <aspect/geometry_model/interface.h>
 #include <aspect/coordinate_systems.h>
```

Comment:
utilities.h and geometry_model/interface.h included each other, which was causing problems